.circleci/config.yml

```diff
@@ -11,22 +11,80 @@ commands:
       - run:
           name: Setup gradle.properties
           command: cp gradle.properties-example gradle.properties
+  yarn-install:
+    steps:
+      - restore_cache:
+          name: Restore Yarn Cache
+          keys:
+            - yarn-i18n-v4-cache-v{{ .Environment.CACHE_TRIGGER_VERSION }}-job-{{ .Environment.CIRCLE_JOB }}-{{ checksum "libs/gutenberg-mobile/yarn.lock" }}
+      - run:
+          name: Yarn Install
+          working_directory: libs/gutenberg-mobile
+          command: yarn install --frozen-lockfile --prefer-offline --network-concurrency 1
+      - save_cache:
+          name: Save Yarn Cache
+          key: yarn-i18n-v4-cache-v{{ .Environment.CACHE_TRIGGER_VERSION }}-job-{{ .Environment.CIRCLE_JOB }}-{{ checksum "libs/gutenberg-mobile/yarn.lock" }}
+          paths:
+            - libs/gutenberg-mobile/node_modules
+            - libs/gutenberg-mobile/i18n-cache/data
+  checkout-submodules:
+    steps:
+      - run:
+          name: Checkout submodules
+          command: git submodule update --init --recursive
+  checkout-gutenberg-mobile-submodule-only:
+    steps:
+      - run:
+          name: Checkout gutenberg-mobile submodule (no recursive)
+          command: git submodule update --init
+  yarn-bundle-android:
```

Comment:
Why do we need that? For generating assets?

Reply:
I think that's right. We're generating a fresh JS bundle so the ci build includes any changes.

Reply:
Hmmm, so we still will have the bundle in `gutenberg-mobile` and we will create the fresh one on the CI? So if you run locally WordPress-Android app w/o packager then you will have the bundle from `gutenberg-mobile/bundle` but the CI will generate a new one? Which JS bundle will be used in the APK generated by peril?

Reply:
I believe Android is never relying on having the bundle committed to gutenberg-mobile. Instead, we're always generating a new bundle if we're not using the packager, both locally and on CI. Obviously this is a different approach to what WPiOS does.

Reply:
Ok got it :) So, in that case, it's pointless to have the android bundle here - https://github.com/wordpress-mobile/gutenberg-mobile/tree/develop/bundle/android, right?

Reply:
I think that's right.